ktor-core/src/org/jetbrains/ktor/features/CallLogging.kt

```diff
@@ -35,8 +38,9 @@ class CallLogging(private val log: Logger, private val monitor: ApplicationMonit
         override val key: AttributeKey<CallLogging> = AttributeKey("Call Logging")
         override fun install(pipeline: Application, configure: Configuration.() -> Unit): CallLogging {
             val loggingPhase = PipelinePhase("Logging")
-            Configuration().apply(configure)
-            val feature = CallLogging(pipeline.log, pipeline.environment.monitor)
+            val configuration = Configuration()
+            configuration.apply(configure)
```

Comment:
Minor: could be one-liner `val configuration = Configuration().apply(configure)`